### ISSUE TITLE
incorporate pyGSM into workflow

### ISSUE THREAD

**Issue by joshkamm** [ZimmermanGroup/py-conformational-sampling#45]
*   [x] create ligands for example in molecular editor
*   [x] Write a test for a reductive elimination reaction
*   [x] Try to have pyGSM work even when fragments aren't entirely contiguous atom indices. This was effectively fixed by debugging using the optimized structure and passing that to pyGSM
*   [x] Have pyGSM not kill strings because the first node is the highest in energy - this is likely no longer an issue now that we're switching from single ended to double ended after growth phase
*   [x] We edited se_gsm.py line 162 to increase the number of opt_steps in the string optimization from 3 to the growing phase default which is currently 10. It would be nice if pyGSM could make this configurable - update: Khoi suggested that this probably would cause other issues so we reverted the change.
*   [x] Update the pyGSM figure output so that it puts things in kcal/mol instead of kJ/mol - this is tied to the xyz writing issue
*   [x] Depends on #48 
*   [x] Debug the weird resource usage behavior when I test that doesn't occur when Soumik tests. A couple things I could try include using the stko way of calling xTB and trying mopac. I could also create a github issue for xTB to ask about why xTB might be behaving differently
*   [x] Update flowchart to include GSM section
*   [x] Try a fresh build and see how things work with that
*   [x] Update documentation
*   [x] parameterize binding tests
*   [x] Look through pull request changes
*   [x] Increment version number
*   [x] Move remaining todo items to new issue

**Comment by joshkamm** [comment]
test comment
